### DISCUSSION TITLE
MP4 files as Live Photo

### DISCUSSION THREAD

**Discussion by ThoBur** [TomoyukiAota/photo-location-map#406] [Ideas]
Hi,

I have a question. I have a folder with many recorded GoPro mp4 files. I extracted a jpg (first image) from each mp4 file and geotagged it using the exif gps location from the mp4 file. The file name and location are the same as the mp4 file. Only the extension is different.

It would be nice if the mp4 file is displayed like MOV Live Photo. Is that possible? Theoretically, only the mp4 extension would have to be added, right?

Thank you for your great tool.

Thomas

**Comment by ThoBur** [comment]
Thank you so much for v1.6.0. Works like a charm.

**Reply by TomoyukiAota** [reply]
Hi Thomas,

Sorry for the late reply. I noticed this discussion a week ago.
Recently I was working on v1.6.0, so the feature can be implemented just the day before the release of v1.6.0.
I'm glad that you liked it.

Tomoyuki
